### DISCUSSION TITLE
telegram - api

### DISCUSSION THREAD

**Discussion by tvideo** [totumonline/totum-ru-issues-and-discussions#299] [ОШБК!] [answered]
Ошибка при работе с ботом телеграмма. Все работало, сейчас почему-то выдает ошибку:
Could not resolve host: api.telegram.org

Функция getFromScript; Линия кода getfs.; Функция jsonExtract; Линия кода extr.; Функция linkToDataJson; Функция exec; Функция jsonCreate; Линия кода viewJson.; Функция linkToDataText; action поле check_remote таблицы ttm__remotes id 1

Вроде бы все проверил, если просто отправить get, скопировав в строку, бот отвечает. С тотума - ошибка. В чем может быть причина?
Спасибо

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Could not resolve host: api.telegram.org - этот хост недоступен из вашего участка сети. Это может быть неработоспособность телеграма или ошибка маршрутизаторе вашей или магистральной сети

**Reply by tvideo** [reply]
но если из строки браузера работает, то возможно причина в хостинге, на котором размещен тотум?

**Reply by tvideo** [reply]
Проблема была на стороне хостинга
